--- origsrc/ggz-gtk-client-0.0.14.1/configure.ac	2008-02-14 12:26:33.000000000 -0600
+++ src/ggz-gtk-client-0.0.14.1/configure.ac	2008-06-05 23:25:01.203125000 -0500
@@ -58,7 +58,7 @@
 
 dnl Check for NLS
 dnl ============================================================
-AM_WITH_NLS
+AM_NLS
 AC_DEFINE_UNQUOTED([PACKAGE_LOCALE_DIR], ["${prefix}/share/locale"],
                    [The location to put message catalogs])
 
--- origsrc/ggz-gtk-client-0.0.14.1/ggz-gtk/Makefile.am	2008-01-05 13:03:36.000000000 -0600
+++ src/ggz-gtk-client-0.0.14.1/ggz-gtk/Makefile.am	2008-06-05 23:25:01.218750000 -0500
@@ -74,7 +74,7 @@
 
 libggz_gtk_la_LIBADD = $(GTK_LIBS) $(LDADD) $(LIBS) $(INTLLIBS) \
 			$(LIB_GGZCORE) $(LIB_GGZ) $(LIB_GETTEXT)
-libggz_gtk_la_LDFLAGS = $(GGZCORE_LDFLAGS) $(LIBGGZ_LDFLAGS) -version-info 1:0:0
+libggz_gtk_la_LDFLAGS = $(GGZCORE_LDFLAGS) $(LIBGGZ_LDFLAGS) -version-info 1:0:0 -no-undefined
 
 #if WIN32
 #ggz_gtk_SOURCES += ggz-gtk-icon.rc
